README.md

```diff
@@ -1,54 +1,90 @@
+You are not allowed to collaborate during the Sprint Challenge. However, you are
+encouraged to follow the twenty-minute rule and seek support from your PM and
+Instructor in your cohort help channel on Slack. Your work reflects your
+proficiency in Testing and your command of the concepts and techniques in the
+Introduction to Automated Testing, Testing React Applications & Testing Web APIs
+modules.
 
 You have three hours to complete this challenge. Plan your time accordingly.
 
 ## Commits
 
-Commit your code regularly and meaningfully. This helps both you (in case you ever need to return to old code for any number of reasons and your project manager.
+Commit your code regularly and meaningfully. This helps both you (in case you
+ever need to return to old code for any number of reasons and your project
+manager.
 
 ## Description
 
-In this challenge use `Test Driven Development` to build a RESTful API using Node.js and Express to create and list _games_. **Data can be stored in memory using a simple JS array**. No need to keep track of incrementing `id`s for this project's MVP, that is part of the Stretch Problem.
+In this challenge use `Test Driven Development` to build a RESTful API using
+Node.js and Express to create and list _games_. **Data can be stored in memory
+using a simple JS array**. No need to keep track of incrementing `id`s for this
+project's MVP, that is part of the Stretch Problem.
 
 ## Self-Study/Essay Questions
 
-Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
+Demonstrate your understanding of this week's concepts by answering the
+following free-form questions. Edit this document to include your answers after
+each question. Make sure to leave a blank line above and below your answer so it
+is clear and easy to read by your project manager.
 
-1. In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
-1. What is the point of `Test Driven Development`? What do you think about this approach?
-1. Mention three types of automated tests.
+1. In Jest, what are the differences between `describe()` and `it()` globals,
+   and what are good uses for them?
+
+   - describe() breaks your test suite into components.
+   - it() is where you perform individual tests.
+
+2. What is the point of `Test Driven Development`? What do you think about this
+   approach?
+
+   - Test pieces of code even before you can test it manually because you're
+     only working in a feature or the code isn't ready for the first test.
+   - I like the idea of test my code before somebody else makes changes that can
+     affect it.
+
+3. Mention three types of automated tests.
+   - Mocha, Jasmine, Jest
```

Comment:
Great answers! The staight answers without any fluff is nice. Straight and to the point!